src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -3345,6 +3305,28 @@ export class StudyViewPageStore {
     updateChartStats() {
         this.initializeChartStatsByClinicalAttributes();
 
+        if (!_.isEmpty(this.molecularProfiles.result)) {
```

Comment:
should updageChartsStats be an action?  I mean, maybe it's called only be things which are actions themselves but seems like it makes a lot of state updates so maybe just tag it as action too?

Reply:
there is already action annotation to it